lib/tasks/link_governments.rake

```diff
@@ -0,0 +1,13 @@
+namespace :governments do
+  task relink: :environment do
+    Edition.where(state: %w[draft published]).each do |edition|
+      presenter = PublishingApiPresenters.presenter_for(edition)
+      links = presenter.links
+      if links[:government].present?
+        print '.'
+        Services.publishing_api.patch_links(presenter.content_id, links: links.slice(:government), bulk_publishing: true)
```

Comment:
I would suggest splitting this line up into multiple lines just because it's a bit long, something like:

```rb
Services.publishing_api.patch_links(
  presenter.content_id,
  links: links.slice(:government),
  bulk_publishing: true
)
```